skins/oasis/js/Tracking.js

```diff
@@ -583,12 +536,11 @@ jQuery(function($){
 				return;
 			}
 
-			switch(id) {
-				case 'wall-notifications-markasread-this-wiki':
-				case 'wall-notifications-markasread-all-wikis': {
-					label = id;
-					break;
-				}
+			switch (id) {
+			case 'wall-notifications-markasread-this-wiki':
```

Comment:
And again? :) The examples in the js guidelines show them as indented: https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#switch-statements. Is there a reason all of these are not?